src/app/gallery/components/public-archives/public-archives.component.ts

```diff
@@ -0,0 +1,36 @@
+/* @format */
+import { Component, OnInit } from '@angular/core';
+import { ArchiveVO } from '@models/index';
+import { AccountService } from '@shared/services/account/account.service';
+import { Router } from '@angular/router';
+import { MessageService } from '../../../shared/services/message/message.service';
+
+@Component({
+  selector: 'pr-public-archives',
+  templateUrl: './public-archives.component.html',
+  styleUrls: ['./public-archives.component.scss'],
+})
+export class PublicArchivesComponent implements OnInit {
```

Comment:
Although I know it's difficult to rename components in Angular, I really think this should have a different name. The name "PublicArchivesComponent" isn't very meaningful since we have a lot of code dealing with Public Archives in the web-app. Maybe something like "MyPublicArchivesComponent" or "PublicArchivesListComponent" could be better names for this. From my initial skimming of the code I wasn't able to tell at a glance whether this was indeed a new component or just an old component we had refashioned since the name was so generic.